### PR TITLE
Prevent local time signature paste crash

### PR DESCRIPTION
**Pull request by miiizen**
Resolves: #18940 

This resolves the crash reported above and fixes the subsequent unreliable pasting behaviour.

### CODE REVIEW

**Review by oktophonie — APPROVED** [review]

File: src/engraving/dom/durationelement.cpp [hunk 1]
```diff
@@ -116,9 +116,16 @@ float DurationElement::timeStretchFactor() const
 //   actualTicks
 //---------------------------------------------------------
 
+Fraction DurationElement::actualTicks(Fraction& tick) const
```
Comment (cbjeukendrup) [inline src/engraving/dom/durationelement.cpp:119]: If you make this `const Fraction&`...

File: src/engraving/dom/durationelement.cpp [hunk 2]
```diff
@@ -116,9 +116,16 @@ float DurationElement::timeStretchFactor() const
...
+{
+    // Use when tick() is unreliable, for example when pasting
+    return globalTicks() / staff()->timeStretch(tick);
+}
+
 Fraction DurationElement::actualTicks() const
 {
-    return globalTicks() / staff()->timeStretch(tick());
+    Fraction t(tick());
+    return actualTicks(t);
```
Comment (cbjeukendrup) [inline src/engraving/dom/durationelement.cpp:128]: ...you can simplify this to `actualTicks(tick())`.

File: src/engraving/dom/durationelement.cpp (hunk 1 above)
Comment (cbjeukendrup) [inline src/engraving/dom/durationelement.cpp:119]: Also, what do you think about naming this method `actualTicksAt`? That might make its meaning slightly clearer, and also the meaning of the `tick` parameter.

**Comment by zacjansheski** [comment]
Tested on MacOS 14, Windows 11, Ubuntu 22.04.3. Approved
https://github.com/musescore/MuseScore/issues/18940 FIXED

**Pull request merged**